src/main/webapp/index.html

```diff
@@ -8,6 +8,7 @@
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
     <script src="https://www.gstatic.com/charts/loader.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <script src="videocontext.js"></script>
```

Comment:
Let's change the file name to video_context.js or video-context.js per:
https://docs.google.com/document/d/1gT_i7KlEyMELecaVo98aBJWP8yiKtAirY9-oTcQCfZM/edit?usp=sharing